packages/cf-component-card/src/CardToolbarLink.js

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ isActive }) => ({
+  position: 'relative',
+  display: 'inline-block',
+  padding: '0.73333rem 2em 0.73333rem 0.5rem',
+
+  '&::after': {
+    content: "'\\f003'",
+    fontFamily: '"cloudflare-font"',
```

Comment:
We should probably use [this](https://github.com/cloudflare/cf-ui/pull/199/files#diff-642d1968224872e5886c42acd10d99b5R120) everywhere instead of relying on "outside" font face definition.

To make it easier, we could write a simple plugin that would replace:

```
createComponent({} => ({
  customFont: 'cloudflare'
}));
```

by 

```
createComponent({} => ({
  fontFace: {
    fontFamily: 'Cloudflare Icons',
    src: [`data:application/x-font-ttf;base64,AAAABBBBBCCCC...`]
  }
}));
``` 

Out of scope for this PR. Just an idea.